File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.cc
#include<iostream>
#include<math.h>
using namespace std;

int main()
{
	ios::sync_with_stdio(0);
	cin.tie(nullptr);
	string a, b;
	cin >> a >> b;
	int A[26] = { 0 }, B[26] = { 0 };

	for (auto& x : a) A[x - 'a']++;
	for (auto& s : b) B[s - 'a']++;

	int ans = 0;
	for (int i = 0; i < 26; i++) ans += abs(A[i] - B[i]);

	cout << ans;
	

	return 0;
}

File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.java
import java.io.*;
import java.util.*;

class Main {
	public static void main(String[] args) throws Exception {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		String a = br.readLine();
		String b = br.readLine();

		int[] cnt = new int[26];
		for(char c : a.toCharArray()) cnt[c - 'a']++;
		for(char c : b.toCharArray()) cnt[c - 'a']--;

		int ans = 0;
		for(int k : cnt) ans += Math.abs(k);
		
		System.out.print(ans);
	}
}